src/io/files.js

```diff
@@ -1249,35 +1249,71 @@ p5.PrintWriter = function(filename, extension) {
    * @method write
    * @param {Array} data all data to be written by the PrintWriter
    * @example
-   * <div class="norender notest">
+   * <div>
    * <code>
-   * // creates a file called 'newFile.txt'
-   * let writer = createWriter('newFile.txt');
-   * // write 'Hello world!'' to the file
-   * writer.write(['Hello world!']);
-   * // close the PrintWriter and save the file
-   * writer.close();
+   * function setup() {
+   *    createCanvas(100, 100);
+   *    background(234, 40, 95);
+   *    button = createButton('SAVE FILE');
+   *    button.style('font-size', '8px');
+   *    button.position(21, 40);
+   *    button.size(60, 20);
+   *    button.mousePressed(createFile);
+   * }
+
+   * function createFile() {
+   *    // creates a file called 'newFile.txt'
+   *    let writer = createWriter('newFile.txt');
+   *    // write 'Hello world!'' to the file
+   *    writer.write(['Hello world!']);
+   *    // close the PrintWriter and save the file
+   *    writer.close();
+   * }
    * </code>
    * </div>
-   * <div class='norender notest'>
+   * <div>
    * <code>
-   * // creates a file called 'newFile2.txt'
-   * let writer = createWriter('newFile2.txt');
-   * // write 'apples,bananas,123' to the file
-   * writer.write(['apples', 'bananas', 123]);
-   * // close the PrintWriter and save the file
-   * writer.close();
+   * function setup() {
+   *    createCanvas(100, 100);
+   *    background(25, 162, 209);
+   *    button = createButton('SAVE FILE');
+   *    button.style('font-size', '8px');
+   *    button.position(21, 40);
+   *    button.size(60, 20);
+   *    button.mousePressed(createFile);
+   * }
+
+   * function createFile() {
+   *    // creates a file called 'newFile2.txt'
+   *    let writer = createWriter('newFile2.txt');
+   *    // write 'apples,bananas,123' to the file
+   *    writer.write(['apples', 'bananas', 123]);
+   *    // close the PrintWriter and save the file
+   *    writer.close();
+   * }
```

Comment:
it's great to have a complete example here! but can you add it as a second example and keep the original as well?